web/javascript/torrent-row.js

```diff
@@ -107,6 +107,14 @@ TorrentRendererHelper.formatDL = function (t) {
     return '↓ ' + Transmission.fmt.speedBps(t.getDownloadSpeed());
 };
 
+TorrentRendererHelper.formatETA = function(t)
+{
+        var eta = t.getETA();
+        if (eta < 0 || eta >= (999 * 60 *60))
```

Comment:
Please stick to current coding style. In particular, even one-line statements have curly braces around them. There's also .jsbeautifyrc in the root of this repository (controlling indentation etc.)